spec/spec_helper_acceptance.rb

```diff
@@ -127,6 +127,7 @@ def device_model
         # install puppet-resource_api on to the server
         on(host, 'puppetserver gem install puppet-resource_api --no-ri --no-rdoc')
         on(host, 'puppet module install puppetlabs-resource_api')
+        on(host, 'puppet module install puppetlabs-netdev_stdlib')
```

Comment:
https://github.com/willmeek/cisco_ios/blob/9c331d4e6737a58104cfe6e4491645af4a88650f/spec/spec_helper_acceptance.rb#L34 should do all the module installations needed.

Reply:
Removed